akka-actor/src/main/scala/akka/pattern/BackoffOptions.scala

```diff
@@ -68,14 +69,28 @@ object Backoff {
    * @param randomFactor after calculation of the exponential back-off an additional
    *   random delay based on this factor is added, e.g. `0.2` adds up to `20%` delay.
    *   In order to skip this additional delay pass in `0`.
+   * @param replyWhileStopped The message that the supervisor will send in response to all messages while
+   *   its child is stopped.
+   */
+  def onFailure(
+    childProps:        Props,
+    childName:         String,
+    minBackoff:        FiniteDuration,
+    maxBackoff:        FiniteDuration,
+    randomFactor:      Double,
+    replyWhileStopped: Option[Any]): BackoffOptions =
```

Comment:
I think it's enough to have this in `BackoffOptions.withReplyWhileStopped` and then no need to duplicate those methods.